website/blog/2023-10-26-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Release Notes - Podman Desktop 1.5
+description: Podman Desktop 1.5 has been released!
+slug: podman-desktop-release-1.5
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing a new onboarding feature that we hope will earn your seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Easy setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.1**: [Podman 4.7.1](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: ...
+- **Environment file support**: ...
+- **Enhancements to the Settings area**: ...
+- **Improved user experience for state changes**: ...
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
+
+Podman Desktop 1.5 is now available. [Click here to download it](/downloads)!
+
+**CUTE SELKIES WILL GO HERE JUST YOU WAIT**
+
+<!-- EXAMPLE -->
+<!-- ![Podman-desktop-1-1-hero](img/podman-desktop-release-1.1/podman-desktop-release-1.1.png) -->
+
+---
+
+## Release Details
+
+<!-- DESCRIBE MAJOR CHANGES, INCLUDE IMAGES / VIDEO IF APPLICABLE -->
+
+<!-- EXAMPLE -->
+<!-- ### Podman v4.5.1 -->
+<!-- Podman Desktop 1.1 moves up to [Podman 4.5.1](https://github.com/containers/podman/releases/tag/v4.5.1). -->
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+**Mo note: I have to fill this in more**
+
+- Notification registry for notifications displayed on the dashboard [#4333](https://github.com/containers/podman-desktop/pull/4333)
+
+### Command Palette
+
+A new, search-driven command palette is now available to enable quick access to various commands available across Podman Desktop. You can try this new tool out by hitting the F1 key in the Podman Desktop UI. [#4081](https://github.com/containers/podman-desktop/pull/4081) &amp;&amp; [#3979](https://github.com/containers/podman-desktop/pull/3979)
```

Comment:
Could remove "in the Podman Desktop UI", it's kinda obvious.

Reply:
done!